cmake/GenerateTestSchedule.cmake

```diff
@@ -0,0 +1,37 @@
+function(generate_test_schedule OUTPUT_FILE)
+  set(options)
+  set(oneValueArgs GROUP_SIZE)
+  set(multiValueArgs TEST_FILES SOLO)
+  cmake_parse_arguments(_generate "${options}" "${oneValueArgs}"
+                        "${multiValueArgs}" ${ARGN})
+
+  list(SORT _generate_TEST_FILES)
+  file(REMOVE ${OUTPUT_FILE})
+
+  # We put the solo tests in the test file first
+  foreach(_solo ${_generate_SOLO})
+    string(REGEX REPLACE "(.+)" "\\1.sql" _test ${_solo})
```

Comment:
Ok, I have no idea why solo tests must not have the `.sql` extension and the rest must have the extension. Can we fix this maybe?